Mod0/computer-setup.md

```diff
@@ -0,0 +1,138 @@
+
+- Please create a free account with Loom using your Turing email address.
+- [Bookmark this webpage on Chrome](https://support.google.com/chrome/answer/188842?hl=en&co=GENIE.Platform%3DDesktop), so that you can easily find it again later.
+- If you've never used Loom before, [watch this video](https://youtu.be/iugmJe8uDLE) to familiarize yourself with this tool.
+
+<br>
+
+### Install Zoom
+
+Zoom is the primary tool for classroom time, study halls, and community building at Turing.
+
+**Action Steps:**
+- [Download and install Zoom](https://zoom.us/download/).
+- If you haven't already, create an account with Zoom.
+- [Update your name in your profile](https://support.zoom.us/hc/en-us/articles/201363203-My-Profile) following this pattern: First Name, Last Initial, pronouns, Program - `Megan M (she/her), Launch`.
+- [Learn how to raise your hand](https://support.zoom.us/hc/en-us/articles/205566129-Raise-Hand-In-Webinar).
+- Give your Zoom account the following permissions: screen sharing, microphone, camera.
+
+<br>
+
+### Snap your Windows
+
+Windows 11 offers a feature to position multiple windows on the screen. Known as Snap Windows, this feature lets you precisely control how and where a window is located by selecting a predefined layout. By snapping each window into a specific place on your desktop, you can more easily juggle multiple windows.
+
+The best part is that there's nothing to install! [Learn more about Snap Windows](https://support.microsoft.com/en-us/windows/snap-your-windows-885a9b1e-a983-a3b1-16cd-c531795e6241) and practice arranging windows on your screen using different layout configurations.
+
+<br>
+
+## That's all folks! 
+To verify that all steps are completed and installations are working as expected, complete the steps below. If you do not get the expected output for any of the following steps, go back to that step in the computer setup and carefully follow all of the steps a second time. 
+1. Quit out of all other applications and close any extra Chrome windows. You can verify that Chrome is running by looking at your dock at the bottom of your screen. The active application (Chrome) will have a gray bar under the icon. Other applications that are currently running will have a small gray dot under each icon.
+1. **Chrome:** In your Chrome window, open a new tab. At the very top under the address bar, you should see at least four bookmarked pages, [GitHub](https://www.github.com), [Replit](https://replit.com/~), [Trello](https://trello.com/), and [Loom](https://www.loom.com/looms/videos). Open each bookmark in its own tab. Close each tab by clicking the `X` attached to each tab.
```

Comment:
Aha - I get it now 🤦‍♀️ 

I was very confused about the bookmarks, so I think we should be more specific.  Like `create a bookmark for 'github.com' by following [these steps](the steps for bookmark creation)`

Reply:
You are absolutely right.